packages/commonwealth/client/scripts/views/pages/CommunityManagement/CommunityProfile/CommunityProfileForm/CommunityProfileForm.tsx

```diff
@@ -107,12 +107,17 @@ const CommunityProfileForm = () => {
         bannerText: values.communityBanner,
       });
 
+      let customStages = values.customStages;
+      if (customStages && customStages.includes("'")) {
+        customStages = customStages.replaceAll("'", '"');
```

Comment:
a very minor edge case here if a stage name has `'`, ex: `In-Discussion's` -- not sure if we have stage names with special chars

Reply:
Do you have any suggestions for validation here? Maybe we should require double quotes around strings so that apostrophes become valid? Not sure how we would validate that though (probably some crazy regex).

Reply:
Ok so after checking it seems there are no existing apostrophes in `custom_stages` so I have updated client validation to not allow single quotes whatsoever. The fundamental issue here is that we are asking users to format an array to begin with. This is bad UX and should be updated so that users add a single stage at a time which is then rendered in a list (like how we add admins/moderators). That way we can properly format each stage as necessary as they are added. That said, this is out of scope for this ticket so removing single quotes is a reasonable alternative for now.

Reply:
> Ok so after checking it seems there are no existing apostrophes in custom_stages so I have updated client validation to not allow single quotes whatsoever.

didn't knew about this, thanks.